### ISSUE TITLE
Stats: add alt text to avoid accessibility issues

### ISSUE THREAD

**Issue by jeherve** [Automattic/jetpack#7821]
Accessibility validators are flagging the stats tracking smiley as problematic. This will apparently prevent sites from meeting some accessibility compliance requirements.

The user adds:

> I believe it's asking for an "alt text" to be included in that line of code? if you want a screen reader to ignore it, than some kind of code like -- alt=" " might do it.

Ticket: 629611-zen

**Comment by timnolte** [comment]
Was going to do this but after further analysis this is wp.com JavaScript code that is generating the stats smiley. Unless it can be added to the $data array and picked up, perhaps that's what I'm missing but I'm pretty sure this is .com code.

**Comment by jeherve** [comment]
closing as duplicate of #18188.
